ariadne_codegen/client_generators/dependencies/async_base_client.py

```diff
@@ -69,12 +69,14 @@ def get_data(self, response: httpx.Response) -> dict[str, Any]:
 
         return cast(dict[str, Any], data)
 
+    def _convert_value(self, value):
```

Comment:
I missed it earlier, `_convert_value` misses type annotations
```suggestion
    def _convert_value(self, value: Any) -> Any:
```